docs/users-guide/Analyzing-and-Translating-Models.adoc

```diff
@@ -562,12 +540,24 @@ translating constant definitions>>.
 The `-g` option is ignored, because
 the include guard prefix comes from the name of the topology.
 
-==== Other Definitions
+==== Types, Ports, and Components
+
+{cpp} code generation for types, ports, and components works similarly
+to the corresponding code generation from XML.
+This code generation is described
+in the https://github.com/nasa/fprime/blob/master/docs/UsersGuide/guide.md[F 
+Prime User's Guide].
+
+==== Component Implementation Templates
 
-We are phasing in direct {cpp} code generation for types, ports, and
-components.
-Currently we have implemented direct {cpp} code generation for types
-(arrays, structs, and enums) and ports.
+`fpp_to_cpp` has an option `-t`.
+When you run `fpp_to_cpp` with this option, it does not
```

Comment:
I think this should be `fpp-to-cpp` instead of  `fpp_to_cpp`.

Reply:
Good catch, fixed!